1.data-exploration/scripts/nbconverted/test-train_split.py

```diff
@@ -8,35 +8,27 @@
 import pathlib
 import numpy as np
 import pandas as pd
-sys.path.insert(0, "./0.data-download/scripts/")
+sys.path.insert(0, "../0.data-download/scripts/")
 from data_loader import load_data
-import matplotlib.pyplot as plt
-from matplotlib.pyplot import figure
-from sklearn.decomposition import PCA
-from tensorflow import keras
-
-from tensorflow.keras.models import Model, Sequential
-import seaborn
-import random as python_random
-import tensorflow as tf
+from sklearn.model_selection import train_test_split
 
 
 # In[2]:
 
 
-# load the data 
-data_directory = "./0.data-download/data/"
+# load all of the data 
+data_directory = "../0.data-download/data/"
 dfs = load_data(data_directory, adult_or_pediatric = "all")
 dependency_df = dfs[1]
 sample_df = dfs[0]
 
 
-# In[24]:
+# In[3]:
 
 
-# searching for nulls
-nan_rows  = dependency_df[dependency_df.isna().any(axis=1)]
-nan_rows
+# assign age_categories and sex columns to the dependency dataframe as a single column
+presplit_dependency_df = dependency_df.assign(age_and_sex = sample_df.age_categories.astype(str) + "_" + sample_df.sex.astype(str))
```

Comment:
ah! One thing to confirm here: Right now you don't have a definitive check that the dependency rows are exactly aligned with the sample info file.

How can you confirm that they are the same order?